src/ast/mod.rs

```diff
@@ -892,7 +895,11 @@ impl fmt::Display for Statement {
                         Sink => "SINKS",
                         Index => unreachable!(),
                     }
-                )
+                )?;
+                if let Some(like) = like {
+                    write!(f, " LIKE '{}'", value::escape_single_quote_string(like),)?;
```

Comment:
there is a trailing `,` in this `write!` macro (rustfmt doesn't add or remove any tokens from macros, because they get to define what is significant)